main.cpp

```diff
@@ -76,45 +103,73 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
 } 
 
-/*
- 1)
- */
+ float carCost(float downPayment, float carPrice, int numCars)
+ {
+     ignoreUnused(downPayment, carPrice, numCars = 1);
+     return {};
+ }
 
-/*
- 2)
- */
 
-/*
- 3)
- */
+  char password(char password, char username)
+ {
+     ignoreUnused(password, username);
+     return {};
+ }
+ 
 
-/*
- 4)
- */
 
-/*
- 5)
- */
+  void moveArm(int direction, int distance)
+ {
+     ignoreUnused(direction, distance);
+ }

```

Comment:
- Fix the indentation.  
`void` is not lined up with the left edge of the editor
- [ ] revise